src/standardPronunciation.ts

```diff
@@ -0,0 +1,316 @@
+          currentSyllable.last =
+            currentSyllable.last.length === 1
+              ? ''
+              : (currentSyllable.last = currentSyllable.last[0] as typeof currentSyllable.last);
+        }
+      }
+
+      /*
+      제13항 - 홑받침이나 쌍받침이 모음으로 시작된 조사나 어미, 접미사와 결합되는 경우에는, 제 음가대로 뒤 음절 첫소리로 옮겨 발음한다.
+      제14항 - 겹받침이 모음으로 시작된 조사나 어미, 접미사와 결합되는 경우에는, 뒤엣것만을 뒤 음절 첫소리로 옮겨 발음한다.
+    */
+      const is음가없는자음앞 = currentSyllable.last && nextSyllable?.first === 음가가_없는_자음;
+
+      if (is음가없는자음앞) {
+        const is홑받침 = currentSyllable.last.length === 1;
+        const is쌍받침 = currentSyllable.last.length === 2 && currentSyllable.last[0] === currentSyllable.last[1];
+        const is겹받침 = currentSyllable.last.length === 2 && currentSyllable.last[0] !== currentSyllable.last[1];
+
+        if (
+          nextSyllable.first === 음가가_없는_자음 &&
+          !['ㅇ', ''].includes(currentSyllable.last) &&
+          (is홑받침 || is쌍받침)
+        ) {
+          nextSyllable.first = currentSyllable.last as Syllable['first'];
+          currentSyllable.last = '';
+        } else if (is겹받침) {
+          if (currentSyllable.last[1] === 'ㅅ') {
+            nextSyllable.first = 'ㅆ';
+          } else {
+            nextSyllable.first = currentSyllable.last[1] as Syllable['first'];
+          }
+
+          currentSyllable.last = currentSyllable.last.replace(currentSyllable.last[1], '') as Syllable['last'];
+        }
+      }
+
+      /*
+        제4장 받침의 발음
+        제9항 - 받침 ‘ㄲ, ㅋ’, ‘ㅅ, ㅆ, ㅈ, ㅊ, ㅌ’, ‘ㅍ’은 어말 또는 자음 앞에서 각각 대표음 [ㄱ, ㄷ, ㅂ]으로 발음한다.
+        제10항 - 겹받침 ‘ㄳ’, ‘ㄵ’, ‘ㄼ, ㄽ, ㄾ’, ‘ㅄ’은 어말 또는 자음 앞에서 각각 [ㄱ, ㄴ, ㄹ, ㅂ]으로 발음한다.
+        제11항 - 겹받침 ‘ㄺ, ㄻ, ㄿ’은 어말 또는 자음 앞에서 각각 [ㄱ, ㅁ, ㅂ]으로 발음한다.
+      */
+      const is어말 = currentSyllable.last && !nextSyllable;
+      const is음가있는자음앞 = currentSyllable.last && nextSyllable?.first !== 음가가_없는_자음;
+
+      if (is어말 || is음가있는자음앞) {
+        if (currentSyllable.last in 받침_대표음_발음) {
+          currentSyllable.last = 받침_대표음_발음[currentSyllable.last as keyof typeof 받침_대표음_발음];
+        }
+      }
+    }
+
+    const changedSyllables = disassembleHangul
+      .filter(isNotUndefined)
+      .map(syllable => combineHangulCharacter(syllable.first, syllable.middle, syllable.last));
+
+    for (const { index, syllable } of notHangulPhrase) {
+      changedSyllables.splice(index, 0, syllable);
+    }
+
+    changedHangul.push(joinString(...changedSyllables));
+  }
+
+  return changedHangul.join(' ');
+}
+
+function 음절분해(hangulPhrase: string): {
+  notHangulPhrase: NotHangul[];
+  disassembleHangul: Syllable[];
+} {
+  const notHangulPhrase: NotHangul[] = [];
+  const disassembleHangul = Array.from(hangulPhrase)
+    .filter((syllable, index) => {
+      if (!isHangulCharacter(syllable) || isHangulAlphabet(syllable)) {
+        notHangulPhrase.push({
+          index,
+          syllable,
+        });
+
+        return false;
+      }
+
+      return true;
+    })
+    .map(syllable => disassembleCompleteHangulCharacter(syllable))
```

Comment:
매우 사소하지만, 309번째 줄에 filter구문과 형태를 맞춰줘도 좋을 것 같아요!

```suggestion
    .map(disassembleCompleteHangulCharacter)
```

Reply:
꼼꼼히 봐주셔서 감사해요!
[fix: map 메소드를 축약한다](https://github.com/toss/es-hangul/pull/115/commits/32218655106a116ad2aa94809fffa2600ba3d0e8)